### PR TITLE
Add `didInitialize` to Keycloak typings

### LINKED ISSUE #27040: [keycloak-js] Expose didInitialize as a public method/property
### Description

There is currently no way for a consumer of the keycloak instance to check if it is initialized without either keeping a local state or error handling. 

It would be ideal if the client would expose the already existing property `didInitialize`. 

### Discussion

_No response_

### Motivation

As per the description, an additional motivation would be for React apps that have `StrictMode` enabled in which case effects are run twice and consumers of the library can easily get into a state where they attempt the init flow twice due to race conditions. 

### Details

I believe the implementation would be as simple as:

```
declare class Keycloak {
// Other properties

         /**
	 * Is true if the client is initialized, false otherwise.
	 */
	initialized?: boolean;
```
And then either rename the property in the JS file or expose a getter, whatever makes more sense for the owners.

### PR DESCRIPTION
**Pull request by maapteh**
closes https://github.com/keycloak/keycloak/issues/27040

follow up on https://github.com/keycloak/keycloak/pull/20722 where it was added to JS but not in its d.ts

~~oeps, wait :)~~

### CODE REVIEW

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/libs/keycloak-js/dist/keycloak.d.ts [hunk 1]
```diff
@@ -450,6 +450,13 @@ declare class Keycloak {
 	*/
 	timeSkew?: number;
 
+	/**
+	* Wether the 'Keycloak' instance is initialized or not (you should only
+	* initialize once!). Undefined when we are still initializing, if 
+	* you called init() :)
+	*/
+	didInitialize?: true
```
[the remark this answers is not in the document]
Comment (maapteh) [reply js/libs/keycloak-js/dist/keycloak.d.ts:458]: I think i wanted to do that, but the Keycloak type is used for intialisation as well which means when i type it as non optional suddenly there it also gets required. For me coding in pure js is long time ago so i have to see how.

Thanks for the review btw!

**Comment by maapteh** [comment]
@jonkoops I think i resolved them :) could not find test suite?

File: js/libs/keycloak-js/dist/keycloak.d.ts [hunk 2]
```diff
@@ -450,6 +450,11 @@ declare class Keycloak {
 	*/
 	timeSkew?: number;
 
+	/**
+	* Whether the instance has been initialized by calling `.init()`.
+	*/
+	didInitialize: true
```
Comment (jonkoops) [inline js/libs/keycloak-js/dist/keycloak.d.ts:456]:
```suggestion
	didInitialize: boolean;
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — APPROVED** [review]
LGTM, thanks @maapteh!

**Comment by maapteh** [comment]
@jonkoops do you know when this typing fix wil land, approximately :)

**Comment by jonkoops** [comment]
You can see when things land by looking at the labels on the issue it [is attached to](https://github.com/keycloak/keycloak/issues/27040). So in this case the tag is `release/26.0.0`, which means it will be part of `26.0.0`.

**Comment by maapteh** [comment]
Thanks for the release Jon!

**Pull request merged**